tests/virt_autotest/login_console.pm

```diff
@@ -84,7 +84,15 @@ sub login_to_console {
             #send key 'up' to stop grub timer counting down, to be more robust to select xen
             send_key 'up';
             save_screenshot;
-            send_key_until_needlematch("virttest-bootmenu-xen-kernel", 'down', 10, 5);
+
+            for (1 .. 20) {
+                if ($_ == 10) {
+                    reset_consoles;
+                    select_console 'sol', await_console => 0;
+                }
+                send_key 'down';
+                last if check_screen 'virttest-bootmenu-xen-kernel', 5;
+            }
```

Comment:
I guess this is to stablize login_console when sol console unstability happens. Cool! Would you mind adding some verification run to virtualization ipmi jobs , like https://openqa.nue.suse.com/tests/3661023?

Reply:
Hello @alice-suse, I was trying several times but I am not able to run that test in my environment. From what I can see it seems like our Prague PXE server does not have the required mounts so I would probably need to contact the infra team to fix it. Is it possible for you or someone from your team to just grab my branch and schedule a test run from your side?
![obrazek](https://user-images.githubusercontent.com/1254493/70401539-65fc3000-1a30-11ea-9433-f4c40e4ae861.png)

Reply:
@pdostal , you can use osd env to verify your PR to know whether it works on osd. 
Run "openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9053 https://openqa.nue.suse.com/tests/3661023" on openqa.suse.de server is enough to trigger one verification job. For more, please refer to http://open.qa/docs/#_triggering_tests_based_on_an_any_remote_git_refspec_or_open_github_pull_request.

If you still have problem to do so, please let me know, I will help with it.

Reply:
@alice-suse OK, I finally managed to do it. The [test run] seems to be OK.

```bash
$ openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9053 https://openqa.suse.de/tests/3661023
Created job #3681190: sle-15-SP2-Online-x86_64-Build101.1-gi-guest_developing-on-host_developing-kvm@64bit-ipmi -> https://openqa.suse.de/t3681190
```

[test run]: https://openqa.suse.de/tests/3681190

Reply:
Cool! Thanks for it!